drivers/overlay/overlay.go

```diff
@@ -580,15 +607,36 @@ func (d *Driver) Get(id, mountLabel string) (_ string, retErr error) {
 		} else {
 			lower = newpath
 		}
-		if newlowers == "" {
-			newlowers = lower
+		diffN = 1
+		_, err = os.Stat(dumbJoin(lower, "..", nameWithSuffix("diff", diffN)))
+		for err == nil {
+			lower = lower + ":" + dumbJoin(lower, "..", nameWithSuffix("diff", diffN))
+			l = l + ":" + dumbJoin(l, "..", nameWithSuffix("diff", diffN))
+			diffN++
+			_, err = os.Stat(dumbJoin(lower, "..", nameWithSuffix("diff", diffN)))
```

Comment:
How does this work after `lower = lower + ":" + …` above? Is that really a valid path?

(And does this need to the separate temporary variable `l`, or could it directly append to `relLowers`?)

Reply:
No, it wouldn't have.